frontend/src/app/app-routing.module.ts

```diff
@@ -16,6 +16,7 @@ import { DashboardComponent } from './dashboard/dashboard.component';
 import { LatestBlocksComponent } from './components/latest-blocks/latest-blocks.component';
 import { ApiDocsComponent } from './components/api-docs/api-docs.component';
 import { TermsOfServiceComponent } from './components/terms-of-service/terms-of-service.component';
+import { BisqModule } from './bisq/bisq.module';
```

Comment:
This causes the BisqModule to be pre-loaded and included in the main.js bundle removing the benefits of LazyLoading. The Bisq bundle is 350 kb (and growing) so would like to keep this lazy loaded.

Reply:
I was unable to see any increase in initial bandwidth. I don't think you've properly removed the Bisq module from the main js file to begin with... so if that is expected behavior there is a bug.

1. visit mempool space and check the size of all the JS files downloaded in the sources tab.
2. visit the Bisq page and notice the extra 76kB minified JS file downloaded.
3. visit mempool ninja and check the size of all the JS files. They are not significantly different in size.
4. visit Bisq and notice nothing is downloaded.

Reply:
I saved (Ctrl+S) each webpage. home, then bisq, for both ninja and space.

```
$ du | sort -k2
11148	.
2816	./ninja
2696	./ninja-bisq
2532	./ninja-bisq/トランザクション - mempool - Bisq Explorer_files
2532	./ninja/mempool - Bitcoin Explorer_files
2836	./space
2796	./space-bisq
2632	./space-bisq/トランザクション - mempool - Bisq Explorer_files
2556	./space/mempool - Bitcoin Explorer_files
```

_files directory (all the assets needed to show the site offline) for space is 76 (kB) difference, but the ninja site is less than 1kB difference... meaning the bisq JS is already in the main JS bundle for space as-is.

In which case you have a bug.

Reply:
Due to the bug. This PR actually decreases the JS downloaded... so in a way, it's a bugfix.

The goal would be to decrease space home from 2556 to 2480 and have Bisq be 2556, but right now you're downloading 2 copies of the same JS. This PR only downloads the JS in the main JS file (which currently contains BisqModule on space)